csharp/Facebook.Yoga/Native.cs

```diff
@@ -46,237 +68,237 @@ internal static class Native
             YogaExperimentalFeature feature);
 
         [DllImport(DllName)]
-        public static extern void YGNodeInsertChild(IntPtr node, IntPtr child, uint index);
+        public static extern void YGNodeInsertChild(YGNodeHandle node, YGNodeHandle child, uint index);
 
         [DllImport(DllName)]
-        public static extern void YGNodeRemoveChild(IntPtr node, IntPtr child);
+        public static extern void YGNodeRemoveChild(YGNodeHandle node, YGNodeHandle child);
 
         [DllImport(DllName)]
-        public static extern IntPtr YGNodeGetChild(IntPtr node, uint index);
+        public static extern YGNodeHandle YGNodeGetChild(YGNodeHandle node, uint index);
```

Comment:
You guys beat me to it. Thank you for noticing this one (@woehrl01)  and removing it (@splhack). It could have caused some confusion and lifetime problems. We should similarly remove `YGNodeSetContext` and `YGNodeSetContext` since this functionality is achieved by the `YogaNode::Data` property.